src/Containers/Microsoft.NET.Build.Containers/Tasks/ComputeDotnetBaseImageTag.cs

```diff
@@ -85,37 +85,25 @@ public override bool Execute()
     }
 
      private string? DetermineLabelBasedOnChannel(int major, int minor, string[] releaseLabels) {
-      // this would be a switch, but we have to support net47x where Range and Index aren't available
-        if (releaseLabels.Length == 0)
+        var channel = releaseLabels.Length > 0 ? releaseLabels[0] : null;
+        switch (channel)
         {
-            return $"{major}.{minor}";
-        }
-        else
-        {
-            var channel = releaseLabels[0];
-            if (channel == "rc" || channel == "preview")
-            {
+            case null or "rtm" or "servicing":
```

Comment:
can you add cases for this to [CanComputeTagsForSupportedSDKVersions](https://github.com/dotnet/sdk/blob/79aaa542200480f0dc26d3ee7bd9d2b0364ae424/src/Tests/Microsoft.NET.Build.Containers.IntegrationTests/TargetsTests.cs#L170] please?

Reply:
#done